Felienne/hedy
# Update athena config #665

**Description**

- Remove accidentally hard-coded athena.table from config
- Add AWS_ATHENA_PREPARE_STATEMENT env variable to hold the athena query name 

**Fix for**

Deploying #665 to beta

**How to test**

All AWS setup for this feature is in place for alpha and beta: athena data is imported, prepare statements are created, IAM policies are assigned to the hedy users, environment variables are added. So,
- Deploy to Alpha. Log as an admin and go to /stats. Verify that log search works as expected.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
